### ISSUE TITLE
Accessbility analysis

### ISSUE THREAD

**Issue by rugk** [rugk/offline-qr-code#82]
One may have a deep look on whether stuff for accessibility can be improved. Considering that we use Photon colors and other standard stuff, which has been tested for accessibility already, I hope it should be quite okay.

To do so, you may [have a look at the accessibility inspector](https://www.marcozehe.de/2018/04/11/introducing-the-accessibility-inspector-in-the-firefox-developer-tools/) of Firefox 61 or add-ons such as [axe devtools](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/).

I had a quick look at the settings and they seem to be okay:
![accessibilityqrcode](https://user-images.githubusercontent.com/11966684/40132937-811e7b54-593e-11e8-84e4-e9f77f63a7b8.png)

But because of (WebExtension) limitations in my Firefox 61 instance I could not get it to actually get it to analyze the popup though. And even the options page needs a check with axe e.g.

Also generally it may be better to run the tool outside of the browser: https://github.com/dequelabs/axe-core https://www.axe-core.org/

**Comment by renatavmaraj** [comment]
Hey! I am interested in doing this ticket!

**Comment by rugk** [comment]
Okay so. Please notice this is mostly a ticket for investigation, with work mostly in docs etc., and does not have to result in a PR.

As such, please (also) explain what you tested and how. And what you noticed, of course.

In the end, you may also have a look on how to fix potentially discovered issues. This could then be a PR. 😃

**Comment by rugk** [comment]
BTW https://ffoodd.github.io/a11y.css/errors.html may be useful.

**Comment by rugk** [comment]
@renatavmaraj  If you need any pointers/help or have questions, feel free to ask.

**Comment by rugk** [comment]
Also good to read/have a look is https://a11yfirst.netlify.com/…

@renatavmaraj Are you still interested in it? If not, that's no problem, just add a notice, so I know what the current status of this issue is.

**Comment by Rish-P** [comment]
Can I work on this? I will need a few days to go over this and come up with something solid... that sound good?

**Comment by rugk** [comment]
@Rish-P  Sure and thanks a lot!
